aiscripts/order.assist.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<diff xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="aiscripts.xsd">
+<add sel="//do_if[@value='not $order.requiredskill or (this.combinedskill ge $order.requiredskill)']">
```

Comment:
`(this.combinedskill ge $order.requiredskill)']"`

 GE a real thing is that supposed to be 'get'

Reply:
"greater than or equals"

that was pulled from DeadAir's Tater update